config/locales/de-CH.yml

```diff
@@ -140,7 +145,7 @@ de-CH:
         invalid_date: ungültiges Datum
         not_a_friday: kein Freitag
         not_a_monday: kein Montag
-        not_an_unsigned_integer: sind keine positive Zahlen
+        not_a_positive_currency_amount: sind keine positive Zahlen
```

Comment:
*sind keine positiven Zahlen or is my german broken?

Reply:
Probably yes